src/picongpu/submit/taurus-tud/k80_profile.tpl

```diff
@@ -24,18 +24,18 @@ TBG_queue="gpu"
 TBG_mailSettings="ALL"
 TBG_mailAdress="someone@example.com"
 
-# 2 gpus per node
-TBG_gpusPerNode=`if [ $TBG_tasks -gt 2 ] ; then echo 2; else echo $TBG_tasks; fi`
+# 4 gpus per node
+TBG_gpusPerNode=`if [ $TBG_tasks -gt 4 ] ; then echo 4; else echo $TBG_tasks; fi`
 
-# number of cores to block per GPU - we got 8 cpus per gpu
-#   and we will be accounted 8 CPUs per GPU anyway
-TBG_coresPerGPU=8
+# number of cores to block per GPU - we got 6 cpus per gpu
+#   and we will be accounted 6 CPUs per GPU anyway
+TBG_coresPerGPU=6
 
 # We only start 1 MPI task per GPU
 TBG_mpiTasksPerNode="$(( TBG_gpusPerNode * 1 ))"
 
 # use ceil to caculate nodes
-TBG_nodes="$(( ( TBG_tasks + TBG_gpusPerNode -1 ) / TBG_gpusPerNode))"
+TBG_nodes="$((( TBG_tasks + TBG_gpusPerNode -1 ) / TBG_gpusPerNode))"
```

Comment:
not rly important, I just annotated it as long as other issues where still open.

but since you did not squash the four commits yet: yes pls amend and fix.